gradle/publishing-repos.gradle

```diff
@@ -5,20 +5,11 @@
  * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
```

Comment:
I'm afraid to ask, but should `http://www.gnu.org/licenses/lgpl-2.1.html` be changed to `https://www.gnu.org/licenses/lgpl-2.1.html`?

Reply:
We have the same problem in `main`. Probably worth updating there first, and then backporting to 5.4 and 5.3?

Though that's really not critical, it's just a comment :)